integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCommonMount.java

```diff
@@ -191,26 +215,237 @@ public void testCreateDomainUsingMultipleCommonMounts() {
+
+    // unzip WDT installation file into work dir
+    unzipWDTInstallationFile(dsCMPath1.toString());
+
+    // create image3 with model and wdt installation files
+    createCommonMountImage(dsCMPath1.toString(),
+        Paths.get(RESOURCE_DIR, "commonmount", "Dockerfile").toString(), miiCMImage3);
+
+    // push image3 to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", miiCMImage3, DOMAIN_IMAGES_REPO);
+      assertTrue(dockerPush(miiCMImage3), String.format("docker push failed for image %s", miiCMImage3));
+    }
+
+    // create stage dir for common mount with image4
+    // replace DataSource URL info in the  model file
+    assertDoesNotThrow(() -> replaceStringInFile(Paths.get(modelsPath1.toString(),
+        "/multi-model-one-ds.20.yaml").toString(), "xxx.xxx.x.xxx:1521",
+        "localhost:7001"),"Can't replace datasource url in the model file");
+    assertDoesNotThrow(() -> replaceStringInFile(Paths.get(modelsPath1.toString(),
+        "/multi-model-one-ds.20.yaml").toString(), "ORCLCDB",
+        "dbsvc"),"Can't replace datasource url in the model file");
+
+    // create image2 with model and wdt installation files
+    createCommonMountImage(dsCMPath1.toString(),
+        Paths.get(RESOURCE_DIR, "commonmount", "Dockerfile").toString(), miiCMImage4);
+
+    // push image4 to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", miiCMImage4, DOMAIN_IMAGES_REPO);
+      assertTrue(dockerPush(miiCMImage4), String.format("docker push failed for image %s", miiCMImage4));
+    }
+
+    // create domain custom resource using common mount with original datasource info
+    logger.info("Creating domain custom resource with domain2Uid {0} and common mount image {1} ",
+        domain2Uid, miiCMImage3);
+    Domain domainCR = createDomainResource(domain2Uid, domain2Namespace,
+        WEBLOGIC_IMAGE_NAME + ":" + WEBLOGIC_IMAGE_TAG, adminSecretName, OCIR_SECRET_NAME,
+        encryptionSecretName, replicaCount, "cluster-1", commonMountPath,
+        commonMountVolumeName, miiCMImage3);
+
+    // create domain and verify its running
+    logger.info("Creating domain {0} with common mount image {1} in namespace {2}",
+        domain2Uid, miiCMImage3, domain2Namespace);
+    createDomainAndVerify(domain2Uid, domainCR, domain2Namespace,
+        adminServerPodName, managedServerPrefix, replicaCount);
+    podsWithTimeStamps = getPodsWithTimeStamps(adminServerPodName,
+        managedServerPrefix, domain2Namespace, 2);
+    // check configuration for DataSource
+    int adminServiceNodePort
+        = getServiceNodePort(domain2Namespace, getExternalServicePodName(adminServerPodName), "default");
+    assertNotEquals(-1, adminServiceNodePort, "admin server default node port is not valid");
+    assertTrue(checkDS(adminServerPodName,domain2Namespace,
```

Comment:
can you check the methods in CommonTestUtils if you can use them - checkSystemResourceConfig and others?

Reply:
switched to common util